tests/fixtures/utils.ts

```diff
@@ -45,3 +47,56 @@ export const prefillDatabase = async (
 
   await db.query([...rootModelTransaction.statements, ...modelTransaction.statements]);
 };
+
+/**
+ * Runs a migration by comparing defined models against existing models and applying the differences.
+ *
+ * @param definedModels - The new/updated model definitions to migrate to.
+ * @param existingModels - The current models in the database.
+ * @param enableRename - Whether to enable model renaming during migration (defaults to false).
```

Comment:
```suggestion
 * @param enableRename - Whether to enable model renaming during migration (defaults to `false`).
```

Notpick, but code should IMO ideally be wrapped in backticks in TSDocs.